libs/providers/go-feature-flag-web/src/lib/model.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import { ErrorCode, EvaluationContextValue, FlagValue } from '@openfeature/web-sdk';
+import { ErrorCode, EvaluationContext, EvaluationContextValue, FlagValue } from '@openfeature/web-sdk';
+import { TrackingEventDetails } from '@openfeature/core';
```

Comment:
Is this not exported from `@openfeature/web-sdk`? If not, we should probably do that.

Reply:
Good catch, I have fixed it.